README.md

```diff
@@ -1 +1,183 @@
+
+
+- [live project](#)
+
+## 🚀 Live Video(still in development) <a name="live-video"></a>
+
+- [live video ](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+ - Installation of Git
+ - Installation of NPM
+ - A working browser
+ - Internet connection
+ - A code editor like Visual Studio Code
+ - Installation of Live Server plugging in Visual Studio Code
+ - Any other tool you can think about
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone git@github.com:rachaouldbabaali/awsome-books.git
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd awesome books
+  npm install
+```
+
+### Usage
+
+To use this project on the `index.html` file with your prefered browser
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Shri sakthi**
+
+- GitHub: [@shri-sakthi-v](https://github.com/shri-sakhti-v)
+- Twitter: [@shrisakthi](https://twitter.com/shrisakthi)
+- LinkedIn: [shrisakthi](https://linkedin.com/in/shrisakthi)
+
+👤 **Racha Ouldbabaali**
+
+- GitHub: [@rachaouldbaali](https://github.com/rachaouldbaali)
+- Twitter: [@rachaoulbaali](https://twitter.com/rachaouldbaali)
+- LinkedIn: [rachaouldbaali](https://linkedin.com/in/rachaouldbaali)
+
+👤 **Yashodhi Chathurangi**
+
+- GitHub: [@Yashodhi-Chathurangi](https://github.com/Yashodhi Chathurangi)
+- Twitter: [@YashodhiChathurangi](https://twitter.com/Yashodhi Chathurangi)
+- LinkedIn: [YashodhiChathurangi](https://linkedin.com/in/Yashodhi Chathurangi)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Adding class for further javascript**
+- [ ] **Adding CSS styleing**
+- [ ] **Adding javascript ES6 module**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please give a star to this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse team, coding partners for the amazing collobrating project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENCE) licensed.
+
```

Comment:
- [ ] Hi team, you have done really well :clap: everything is working well , however I cannot yet approve the project because you have a broken MIT license link in your Readme and in order to make sure our Readme is professional you need to fix this issue :+1: